app/controllers/angular_template_controller.rb

```diff
@@ -0,0 +1,5 @@
+# used to serve static angular templates from under app/views/angular_template/
+
+class AngularTemplateController < ApplicationController
```

Comment:
How/where this controller should be used by developers in miq?
Should this controller be inherited/extended by other controllers of applications that wish to use angular templates?
It just a bit confusing that for an Angular template one must use a Ruby controller that has little to do with Angular or html...